docs/src/man/getting_started.md

```diff
@@ -159,14 +159,19 @@ using CSV
 using DataTables, CSV
 ```
 
-Datasets can now be read using
+Datasets can now be read from files using
 ```julia
-CSV.read(...)
+CSV.read(input, DataTable)
```

Comment:
`input` is still not defined. I would add something to the previous paragraph, like "A dataset can now be read from a CSV file at path `input` using". Same for `output`.